docs/conf.py

```diff
@@ -92,9 +92,14 @@
 
 html_static_path = ['_static']
 html_theme = 'furo'
+html_logo = '_static/images/Ansible-Mark-RGB_Pool.svg'
 html_theme_options = {
-    'dark_logo': 'images/Ansible-Mark-RGB_White.svg',
-    'light_logo': 'images/Ansible-Mark-RGB_Black.svg',
+    'light_css_variables': {
+        'color-background-secondary': '#dee3ed',  # for navigation + ToC
```

Comment:
So the only contrasting color to this I've found was `#414141`. But it's dark, so it's not a good fit for the light theme. This means that the logo color needs to be darker to have a good enough contrast.

Reply:
afaik there is no other logo variant other than the black/white ones you had here already.

Reply:
Looks like https://github.com/ansible/logos/ has a few. I think I saw gray, mango, pride. Do you think any of those would work with the light theme?